examples/cms-sitefinity/package.json

```
@@ -0,0 +1,26 @@
+{
+    "private": true,
+    "scripts": {
+        "dev": "next",
+        "build": "next build",
+        "start": "next start",
+        "typecheck": "tsc",
+        "lint-fix": "pnpm prettier --write . && eslint . --ext js,jsx,ts,tsx --fix --max-warnings=0 --config .eslintrc.json --no-eslintrc"
```

```suggestion
        "start": "next start"
```

The example should be minimal and focused.